### ISSUE TITLE
Raycast exception

### ISSUE THREAD

**Issue by derwaldgeist** [vasturiano/react-globe.gl#59]
I got my globe working now, it's looking amazing.

However, quite often if I hover over it or try to drag it, it throws an exception:

![image](https://user-images.githubusercontent.com/3984535/122469121-db997280-cfbc-11eb-93e1-bca9076514a7.png)

After that, the globe won't allow the user to rotate it anymore. It's stuck to its current position and rotation.

What might cause this?

**Comment by derwaldgeist** [comment]
I can reproduce it pretty reliably if
- there is a label at lat = 0, lon = 0
- I hover the mouse over it
- and then move the mouse within the label's dot

**Comment by vasturiano** [comment]
@derwaldgeist I've tried your steps but am unable to replicate the issue unfortunately.

Could you make a simple reduced example on https://codesandbox.io/ that reliably replicates the issue so it can be debugged?

**Comment by maker2002** [comment]
I have the same issue in https://github.com/vasturiano/globe.gl/ this version.
But I can't reproduce it I a certain way, it's quite random...

**Comment by vasturiano** [comment]
@maker2002 thanks for contributing. I'm still unable to replicate this issue. Are you able to do an example on https://condesandbox.io that produces this error, even if just occasionally?

**Comment by derwaldgeist** [comment]
Yep, I agree, if seems to happen at random. I just felt it was more reproducible if I hovered above a label created at 0,0. But also here not always.

**Comment by devtarek** [comment]
I have the same problem and cannot reproduce it reliably, it happens randomly. I am using react-globe.gl with Next.js

<img width="673" alt="err" src="https://user-images.githubusercontent.com/38392158/123271637-56084c00-d501-11eb-8cdf-d8e36c7441b4.png">

**Comment by zenyr** [comment]
While I was tinkering with react-globe.gl (the official example) I also encountered this error three times. It indeed was very random and I also couldn't reproduce it reliably.

**Comment by phocks** [comment]
Can confirm the exact same issue. Seems to happen randomly. Sorry this is not much help in tracking down the root cause.

**Comment by DavidDolyak** [comment]
Same here!

**Comment by phocks** [comment]
FYI:

I managed to fix this by wrapping this else block of code in an `if` to check that positionAttribute was set

I did it in my `node_modules/three/build/three.module.js` which isn't ideal but didn't really know how else to do it.

I just need to remember to do this again if I ever reinstall three via npm or something.

Maybe it needs to be fixed in the three.js repo.

```
if (positionAttribute) {
  const start = Math.max( 0, drawRange.start );
  const end = Math.min( positionAttribute.count, ( drawRange.start + drawRange.count ) );

...
}
```

**Comment by marcotw** [comment]
Hi, I have the same problem, any news about a fix?

**Comment by rwchampin** [comment]
its the draw range......

**Comment by Surefire10** [comment]
> its because we were calling intersectObject on a mesh like Points.....at least for me and the count was messed up

Is it okay to tell me how to fix it? Or prevent it from happening? I don't wanna crash production lol

**Comment by dcalano** [comment]
Maybe this merge request will help?

https://github.com/mrdoob/three.js/pull/28163

**Comment by z1ndabad** [comment]
This is a cursory observation from someone new to both Three and this library, so YMMV. But I've found that I only hit this issue when passing bad data to the globe--i.e., making sure all keys used by accessors were defined before passing data to the globe fixed this error for me.
